src/utils/ResponseFlags.ts

```diff
@@ -1,15 +1,20 @@
-// The Envoy flags can be found here:
-// https://github.com/envoyproxy/envoy/blob/master/source/common/stream_info/utility.cc
+// The Envoy response flags can be found here:
+// https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage
```

Comment:
I don't see how this link is relevant, compared to the old one. The old utility.cc I can see the 2-char codes.. but I don't see those in the new link.

Reply:
Oh, I see.. %RESPONSE_FLAGS%... should probably mention that in the comments since it is half-way down on the page.